Simulator/augments.py

```diff
@@ -0,0 +1,52 @@
+from enum import Enum, auto
+
+
+class Augment(Enum):
```

Comment:
I like that we have an Augment class but I would like it if all of the methods below were part of this class. Similar to how Player is. Maybe we have 2 classes and one class is the enum and one class gives the implementation of the specific augment.